Compilation on OSX is broken(ish)

The include in main.cpp:41
```
#include <xcodebuild/main.cpp>
```
which is used if ```MACOSX``` is defined, does not necessarily exist. I have a custom compiler setup and for me compilation on OSX fails with

```
main.cpp:41:10: fatal error: 'xcodebuild/main.cpp' file not found
```

I get the compilation to finish by changing that part to
```
#include <CoreFoundation/CoreFoundation.h>
int main(int argc, char **argv)
{
```

--- a/main.cpp
+++ b/main.cpp
@@ -38,7 +38,7 @@
 
 #ifdef MACOSX
 #include <CoreFoundation/CoreFoundation.h>
-#include <xcodebuild/main.cpp>
+#include <XCodeBuild/main.cpp>
 int nuvieMain(int argc, char **argv)
 #else
 int main(int argc, char **argv)

